### ISSUE TITLE
New format: gVCF

### ISSUE THREAD

**Issue by slaurie** [edamontology/edamontology#627]
*Instructions:* When making a request, replace the text < between chevrons > after first reading the instructions.  The instructions can be removed. Do not change any other text, and please try to complete as much information as you can (your request will get acted on sooner if you do).

**Format name:**  < gVCF >

**Short description:** < genomic Variant Call Format (gVCF) is a version of VCF that includes not only the positions that are variant when compared to a reference genome, but also the non-variant positions as ranges, including metrics of confidence that the positions in the rang eare actually non-variant e.g. minimum read-depth and genotype quality >

**Parent URI(s):** < http://edamontology.org/format_2921|http://edamontology.org/format_3016 >

*Instructions:* URI(s) of suggested EDAM Format parent(s) delimited by pipe ('|') *e.g.* "http://edamontology.org/format_3547|http://edamontology.org/format_2333". Format concepts normally have two parents: 1) indicating the basic type *e.g.* "Binary format", "Textual format" *etc.* (see http://edamontology.org/format_1915) and 2) indicating the type of data *e.g.* "Image format" (see http://edamontology.org/format_2350).

**exactSynonym(s):** < synonym1 | synonym2 |... >

*Instructions:* Exact synonym(s) delimited by pipe ('|') *e.g.* "png". Other commonly-used terms, acronyms *etc.* by which the concept is referred to.  This can also include capitalisation variations (as in above example) and use of "." prefix.

**Specification:** < URL1 >

*Instructions:* URL to formal specification of the format *e.g.* "http://www.w3.org/TR/PNG/". Provide a link to the official specification of the format (if available), either an XML schema (XSD), or other rigorous & comprehensive syntax specification.

**Documentation:** < https://sites.google.com/site/gvcftools/home/about-gvcf|https://gatk.broadinstitute.org/hc/en-us/articles/360035531812-GVCF-Genomic-Variant-Call-Format>

**Wikipedia:** < https://en.wikipedia.org/wiki/Variant_Call_Format >

**Publication:** < DOI1 | DOI2 |... >

*Instructions:* Citation DOI *e.g.* "10.1038%2Fnbt1031" or URL *e.g.* "https://www.iso.org/standard/29581.html". Specify a DOI of an article (if available) that describes the format, and should be used to cite mentions or usage of the format.  Specify a DOI (if available) otherwise a URL.

**Data represented:** < http://edamontology.org/data_3498 >

*Instructions:* URI(s) of EDAM Data concept(s) the format represents delimited by pipe ('|') *e.g.* "http://edamontology.org/data_2968". Please specify the EDAM Data concept(s) for the type(s) of data represented by the format.  If you are not sure, or if you can't find the Data concept you need, you can use free text *e.g.* "Image data" instead of the URI.

**File extension(s):** < g.vcf|g.vcf.gz >

*Instructions:* File extension(s) in common use delimited by pipe ('|') *e.g.* "png". Please specify all file name extensions that are commonly used.  Do not include period (`.`) character.

**Example(s):** < https://gatk.broadinstitute.org/hc/en-us/articles/360035531812-GVCF-Genomic-Variant-Call-Format >

*Instructions:* URL(s) to examples of the file format delimited by pipe ('|') *e.g.* "http://prodata.swmed.edu/mummals/info/fasta_format_file_example.htm".  Do not give links to Wikipedia.

**Information standard(s):** < URL1 | URL2 |... >

*Instructions:* URL(s) to relevant information standard which the format supports, delimited by pipe ('|') *e.g.* "http://www.psidev.info/miape". Give link(s) to the information standard homepage(s).

**Ontologies used:** < URL1 | URL2 |... >

*Instructions:* URL(s) to ontologies used by this format, delimited by pipe ('|') *e.g.* "http://edamontology.org/".  Give link(s) to the ontology homepage(s).

**Governing organisation(s):** < https://www.ga4gh.org/#/fileformats-team >

*Instructions:* URL(s) to organisation(s) that formally governs the format, delimited by pipe ('|') *e.g.* "http://www.psidev.info/".  Give link(s) to the organisation homepage(s).  

**Media type:** < media_type >

*Instructions:* Media type *e.g.* "image/png". Specify the formal media type (if available) as per https://www.iana.org/assignments/media-types/media-types.xhtml.

**Comment:** < comment >

*Instructions:* Comment to the EDAM developers. Add any comments you like for the EDAM developers to consider.

**Comment by matuskalas** [comment]
Awesome, thank you so much @slaurie!

**Comment by matuskalas** [comment]
We'd like to add _Co-authored-by_ when committing this (unless you object). Would you please write which email address should be used in the commit? (It can also be the [no-reply GitHub email](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address#about-commit-email-addresses) as a private option.) Thanks! 🙏🏽

**Comment by slaurie** [comment]
No problem Matúš, glad to be able to help.
You can use steven.laurie@cnag.crg.eu

**Comment by melibleq** [comment]
Hi @slaurie and @matuskalas. Was this format already submitted as a proposal for a new term in the edam-browser website? If not, would you mind if I did it? Thanks!

**Comment by Leena1666** [comment]
can I work on this ?

**Comment by matuskalas** [comment]
> Was this format already submitted as a proposal for a new term in the edam-browser website? If not, would you mind if I did it?

@melibleq please don't, that would just create a new issue like this one.

@Leena1666 and @melibleq, would you please try - either in parallel or one after the other helping each other - to implement this in the `EDAM_dev.owl` file? Please include it underneath the last EDAM format (with URI in form http://edamontology.org/format_XXXX). You can leave the URI of this new concept as XXXX, it will be fixed right before merging. Please also check the info and links provided in the issue and improve in case it's needed. All the best and giant thanks! 🙏🏽

**Comment by melibleq** [comment]
> > Was this format already submitted as a proposal for a new term in the edam-browser website? If not, would you mind if I did it?
> 
> @melibleq please don't, that would just create a new issue like this one.
> 
> @Leena1666 and @melibleq, would you please try - either in parallel or one after the other helping each other - to implement this in the `EDAM_dev.owl` file? Please include it underneath the last EDAM format (with URI in form http://edamontology.org/format_XXXX). You can leave the numeric 4-digit ID in the URI of this new concept as XXXX, it will be fixed right before merging. Please also check the info and links provided in the issue and improve in case it's needed. All the best and giant thanks! 🙏🏽

Sure, thank you as well! I'll start working on it. @Leena1666, were you able to set up a copy of the repository in your computer? I'm also a beginner here, but I would be glad to try to provide help in case you face any issues.

**Comment by melibleq** [comment]
Here's a question: the Editors Guide [section](https://edamontologydocs.readthedocs.io/en/latest/editors_guide.html#id12) about Formats says file extensions "MUST NOT include period (‘.’), e.g. 'txt' not '.txt'". But this one has periods "in the middle" of it (g.vcf and g.vcf.gz). Should we remove those periods too when editing the `EDAM_dev.owl` file?

**Comment by matuskalas** [comment]
Nope, please leave them in, thanks 🙇🏽‍♂️

**Comment by melibleq** [comment]
I've just made a [draft pull request](https://github.com/edamontology/edamontology/pull/662/files) for this issue. A couple observations:
- Judging by previous Format additions to `EDAM_dev.owl`, OWL attributes `<documentation>` and `<example>` don't seem to be using `rdf:resource=` when we add website URLs, so I have simply provided the corresponding URLs to those attributes.
- I've created a new branch to work on this issue, but it still contains my previous commit (exactly the same one in [this](https://github.com/edamontology/edamontology/pull/661/commits) other pull request). I guess that's not how it's supposed to be. But for now, while we're working on this issue and the previous commit still hasn't been approved, should I leave it like that?

**Comment by matuskalas** [comment]
> You can use [steven.laurie@cnag.crg.eu](mailto:steven.laurie@cnag.crg.eu)

@slaurie would you like to add this email address to your GitHub profile, so that your co-authored commits are attributed to your account? (I think you can keep it "private" on GitHub if you want, but nevertheless it is now visible in the commits. And in this issue😸. If that is a problem, please let me know asap and I'll purge it😉)
